packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -1391,6 +1401,7 @@ export class ColorPicker
     x: number,
     y: number,
     color: Color,
+    sliderType: SliderType,
```

Comment:
`sliderType` only seems to be used to control whether the color's alpha is applied or not. Signature-wise, it might be simpler to:

1. replace the argument with a boolean to apply alpha
2. drop the argument and rely entirely on `color` for this by making sure we update the color's alpha value when passing it to this function

I think 1. seems simpler and avoids extra creating color objects.